principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web, also known as web3. Our challenge is achieving mass adoption while staying true to our principles outlined below.
```

Comment:
`web3` sounds like a product/technology name, that might change or go away. If we want this document to be as timeless as possible, I'd rather remove it from here. Decentralized web or decentralized communications is enough, IMO.

Reply:
Good point! Agree.